projects/elife.yaml

```diff
@@ -1639,25 +1639,20 @@ iiif:
                             path: /lax:26866/elife-26866-fig3-video1.jpg
                             expected: "article/26866 article/26866/videos"
                         encoded-urls:
-                            path: /lax/26866%2Felife-26866-fig3-video1.jpg
+                            path: /lax/26866%252Felife-26866-fig3-video1.jpg
                             expected: "article/26866 article/26866/videos"
                 digest-id:
-                    # does work:
-                    url: "^/digests/([0-9]+).*$"
-                    # doesn't work:
-                    #url: "^/digests/([0-9]+)(?:/|%2F)(?:.+)$"
-                    # also doesn't work:
-                    #url: "^/digests/([0-9]+)(?:/|%2F)"
+                    url: "^/digests/([0-9]+)(?:/|%252F)(?:.+)$"
```

Comment:
Turns out we had encountered this problem before, just hadn't worked out what was happening.